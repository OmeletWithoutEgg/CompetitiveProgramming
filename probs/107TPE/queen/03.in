38
-45763 54965
70035 -53107
87458 -59363
-21121 40440
62423 -69805
-74761 -88237
15171 -45863
29551 -14114
-39125 -65847
-77810 90911
-15950 -9902
-89564 -1786
99678 -80141
-3246 60515
48733 83676
12503 13005
-44105 -65899
65596 -42383
-60628 30748
-56294 -96443
-12406 18985
51012 6684
-9602 -17322
-80954 56376
69318 -53985
72251 57094
-20175 -63953
-51725 -46051
445 -46282
-79199 -17778
44421 -27635
-59558 59558
-97419 -73763
-6322 -48825
-93649 48349
24606 9733
11403 -25502
-58301 57526
