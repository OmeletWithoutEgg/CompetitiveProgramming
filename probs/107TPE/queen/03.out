-11134.24 2517.87
