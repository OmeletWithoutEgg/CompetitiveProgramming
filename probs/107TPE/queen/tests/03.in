1886
-60053 98146
-50865 107014
-59545 95514
-63747 94135
-41145 111067
-57498 92915
-44746 91056
-45877 86597
-50111 106916
-49015 91861
-61064 110374
-48466 96807
-45071 107971
-62719 94109
-47218 90526
-51952 111726
-58067 103380
-54686 82604
-43455 109046
-71441 106107
-49809 110215
-65861 83988
-67136 101495
-58272 97218
-53470 100764
-67144 98688
-45255 104514
-54652 98438
-46724 96669
-64894 102172
-57261 105581
-65578 109438
-47708 87890
-57004 93881
-73511 85316
-56685 96110
-45202 82925
-53705 92280
-51127 101737
-55550 109329
-58292 109961
-70061 104915
-72367 95072
-53918 90643
-52313 99963
-70534 83271
-71284 82296
-44532 99824
-41612 96327
-69015 103208
-59821 104291
-44248 86672
-61012 84828
-69320 108626
-42092 94309
-62372 86128
-70879 102406
-43552 86367
-59703 103813
-50681 92099
-61986 102993
-40676 82721
-55275 89454
-41953 89646
-64653 104433
-41632 96046
-67478 87654
-48175 82409
-52702 87224
-54869 82181
-44147 82861
-48421 111145
-43995 87885
-60639 83530
-68411 97954
-46531 102299
-71815 98281
-72458 90039
-59834 105770
-59323 104422
-49590 89344
-64655 97093
-45703 100367
-65992 106781
-73178 111605
-51652 102879
-70353 97375
-59676 94719
-40991 111105
-72100 83522
-70120 109783
-55725 98354
-56891 103632
-52622 100682
-54412 90291
-68882 91649
-49706 87574
-68185 84082
-62538 101171
-52292 94750
-67300 105230
-67823 93765
-41383 92830
-71464 92522
-67188 89260
-46124 96578
-70384 88348
-61722 82390
-67501 86916
-61394 102704
-66728 108542
-45442 111244
-59843 91960
-46209 90043
-63664 107576
-54573 110962
-47389 94304
-55240 105849
-71843 106365
-50011 101993
-52302 84461
-41492 104552
-51327 87999
-52418 98672
-47266 90660
-58438 96840
-69559 110878
-71494 111260
-68031 111767
-52725 98390
-56617 96451
-49632 100097
-67734 93879
-40673 82302
-57059 100322
-66559 90544
-54888 96230
-69163 94721
-63190 103102
-69835 84317
-43415 99266
-49260 102534
-48442 105667
-68730 105163
-51531 95035
-52245 93321
-54930 96777
-57040 90508
-60326 103808
-43500 90475
-64022 97955
-41604 83839
-44738 106297
-50203 87892
-72077 82379
-71700 99867
-52843 102376
-52701 101560
-53071 94017
-55190 94614
-73036 88627
-67169 86951
-67383 85628
-53216 94577
-54516 109731
-61336 84544
-70938 107981
-60945 101271
-47483 83227
-62153 109478
-52998 105811
-72946 108364
-46900 103831
-42318 106310
-49092 106788
-54936 96628
-59541 97815
-47534 95932
-69544 106425
-65159 82927
-59349 83300
-54978 94965
-52867 97295
-53571 100802
-42474 85671
-56304 109228
-59241 97225
-69741 85026
-44190 95136
-46897 88720
-65589 95306
-51353 89036
-50878 87992
-44156 86328
-44895 92488
-45747 86006
-57529 108506
-69008 110892
-41826 85437
-72979 102056
-70541 108845
-41505 103870
-70185 86806
-68251 102562
-73316 95862
-61685 90334
-61634 97715
-52358 111208
-43018 86770
-63647 108910
-61217 91101
-46533 89834
-42615 93732
-52212 83002
-48115 95815
-46558 88805
-52798 90104
-57678 102748
-46891 102275
-65914 104724
-40787 87540
-61929 91231
-42367 101612
-52253 100332
-68343 111303
-49348 93891
-46484 104854
-56780 100892
-58358 106294
-46063 86071
-52904 86213
-46644 98398
-67813 102172
-66184 97341
-56997 84166
-48879 111196
-43809 104125
-65830 109648
-57425 85613
-58854 83213
-48868 88915
-43755 109235
-52361 90002
-44300 98391
-59625 83951
-54542 98183
-72531 105821
-45535 103768
-43779 91082
-71945 86724
-70939 92548
-71104 94163
-47563 94880
-69648 87072
-42731 86888
-68865 81782
-66096 89305
-51921 109438
-54246 83350
-50557 87786
-67765 96897
-70997 94033
-44980 97601
-64475 90092
-47899 81918
-70189 89042
-49090 107864
-43346 111369
-71021 86309
-46043 91745
-49571 91716
-46340 104640
-61825 99634
-42205 106981
-42067 109626
-40984 85549
-67663 104832
-50734 82463
-65920 110129
-48616 93998
-54139 93425
-46617 86070
-48346 90643
-70937 97231
-55064 88190
-59495 85285
-55020 85751
-46002 96035
-47318 99421
-41865 110144
-63401 93753
-64081 98466
-57577 99289
-64507 99011
-53764 95168
-73578 87435
-44035 109040
-71875 86262
-59501 108180
-54596 98122
-71808 104781
-49996 84876
-47885 103623
-66246 111417
-47742 111441
-61373 96653
-46765 82487
-61862 95985
-42909 99051
-50932 85386
-47675 111060
-57076 95958
-61265 88533
-49010 97144
-58761 95549
-72864 95259
-70450 89239
-57397 100078
-56954 88700
-55373 102148
-48371 89807
-48278 102837
-53875 96935
-43792 99318
-71653 105632
-43109 85663
-61438 95056
-52337 95033
-42188 96542
-68830 84003
-56575 104835
-61008 88223
-48870 110177
-53574 81936
-61024 92807
-71590 87808
-45462 85873
-60733 109600
-44099 102863
-61231 106142
-43112 102757
-46471 104782
-43735 101943
-55188 88301
-68079 100538
-55733 106448
-57939 110669
-47909 98559
-61921 90327
-52353 104740
-45881 104642
-68992 110935
-46773 101846
-42413 98992
-72306 94654
-66086 93165
-57104 93625
-56040 110779
-62415 98495
-72274 87315
-65769 103638
-46126 110243
-72128 108633
-47461 105728
-52105 98470
-43445 100346
-70588 88351
-56705 101497
-53558 88852
-58335 82640
-66248 86946
-51886 83566
-45719 100266
-51712 82835
-64861 105903
-67487 90618
-49369 91272
-60723 85326
-52775 101443
-68440 95651
-58359 110667
-46404 109630
-55866 104231
-52528 96265
-68373 109485
-47668 83473
-44747 108777
-47807 92812
-61453 101862
-49526 101780
-63884 110072
-62337 95078
-61043 92932
-41446 102093
-60432 102150
-57317 109882
-45979 99877
-61538 98745
-51096 88759
-64625 84678
-44607 85426
-66385 99553
-59556 104283
-61229 103468
-72820 84250
-43657 84500
-60888 87505
-58570 95616
-42013 85117
-42561 106412
-57784 111235
-53066 108473
-43223 82062
-67559 93460
-42618 105829
-44039 95340
-47543 86988
-45543 98677
-67255 94425
-46235 82284
-54168 90992
-53449 108963
-57264 87438
-40721 87606
-55019 110885
-68169 92845
-67970 92171
-62704 108913
-42487 105009
-41951 99466
-47594 86804
-52747 102326
-71731 110351
-62869 98523
-69517 100174
-52016 91023
-58047 104662
-46067 103817
-65713 109118
-63636 100141
-59150 91154
-67863 87854
-55479 94690
-49907 85541
-59571 84728
-42728 108346
-56269 93467
-66894 97363
-56592 82596
-42386 91308
-47469 83035
-73384 87776
-62047 104287
-57297 85574
-59091 108845
-50569 102760
-62603 92720
-68729 98859
-58545 94582
-53898 89344
-47598 108587
-49262 103441
-56508 100452
-57647 108047
-53299 98032
-63816 98642
-58142 82056
-70756 108002
-62277 91432
-46787 94739
-70754 97767
-55845 92130
-69454 89580
-45565 102254
-46068 83694
-54066 94344
-51132 94335
-42081 99089
-47871 107744
-64884 83367
-72693 104080
-64914 109457
-61912 86042
-50991 81780
-42456 110503
-51121 97626
-46405 102642
-70798 85359
-51876 90904
-67718 100115
-42313 88911
-68270 108141
-70705 103856
-50045 103697
-71147 92330
-68029 83916
-46178 89897
-53679 106862
-44995 111176
-59767 86702
-72283 91708
-72662 105916
-51666 100634
-73032 108026
-49072 98565
-65626 107198
-51649 88200
-57207 87651
-69355 101502
-63614 106293
-43103 107705
-50366 98954
-68628 101717
-73163 107047
-51112 106913
-62251 89224
-52268 101547
-46442 101650
-48708 93823
-68575 92895
-66825 100241
-54736 94583
-63277 97494
-70359 94409
-70173 92943
-58832 94453
-55044 81812
-58773 105518
-55033 104748
-70330 89590
-59019 111841
-53706 111453
-61893 100413
-48400 93715
-57125 97831
-71660 103370
-73063 82881
-73550 104279
-56196 107647
-60202 95785
-52653 97460
-41231 87736
-73372 83982
-57333 100242
-61899 84625
-59528 104202
-42787 111855
-40856 109517
-40683 92211
-56425 107613
-68741 99391
-72670 89853
-42554 107619
-57584 109060
-67416 110179
-49737 90746
-65440 111181
-61591 108022
-46564 86607
-61524 91827
-58198 88605
-58880 90983
-59772 106278
-67711 110700
-72470 107295
-46520 98850
-46585 91737
-43013 88842
-41157 111856
-54777 111832
-48675 110781
-72480 101167
-73574 85871
-58770 82125
-51755 91921
-41515 105560
-41878 98342
-72755 94135
-51594 83468
-53016 109690
-71611 100190
-53266 86804
-48903 100214
-70269 100454
-61502 88614
-65995 87642
-62875 106208
-48842 86589
-57668 83928
-48684 108003
-42476 103777
-51246 94201
-49412 98119
-69410 83671
-72876 95525
-64983 83759
-49601 103837
-47690 82118
-48860 111110
-66768 105175
-72848 91952
-60178 98804
-72110 97840
-43872 107265
-65735 107767
-52472 91715
-54960 102895
-67973 94198
-57458 100480
-42068 103291
-73288 104120
-42455 99444
-40974 84882
-47038 97268
-57897 94870
-53229 101271
-59162 87770
-57420 82749
-54715 84921
-60520 100616
-72217 108004
-64301 108313
-46486 92407
-68764 84945
-53929 84966
-41786 102255
-55081 96970
-60935 101753
-73133 91183
-67614 102072
-60005 111823
-70602 90833
-51522 111876
-68585 90062
-49898 98853
-49433 91810
-66884 100174
-51950 81934
-45212 95316
-51569 105845
-57270 96503
-53362 105034
-51746 99930
-64259 103094
-54525 93430
-69252 94047
-44427 83867
-54849 86030
-41933 96758
-57866 108794
-49275 85405
-60993 91022
-50905 82181
-62356 94201
-73322 94620
-58945 110760
-44788 107820
-70448 97772
-56019 92420
-61641 85051
-47017 102780
-60599 86946
-43700 102741
-51995 101275
-63865 110877
-58490 96743
-41636 109142
-66671 109196
-55753 96260
-42827 92781
-51218 101781
-42123 111355
-47533 96705
-69397 85345
-61894 105304
-67185 104271
-46323 96496
-52626 97226
-46574 91302
-64205 101566
-65351 109759
-61514 91705
-71516 110694
-42029 108766
-56204 106500
-44908 82604
-47802 110450
-42738 101155
-63714 103833
-47693 103255
-65505 87480
-71413 86828
-53048 84745
-41883 102779
-65119 109505
-50388 108252
-46004 104634
-49272 110791
-50121 106243
-43918 88174
-53608 94414
-48652 107804
-44006 111869
-49449 100685
-71922 100118
-41449 89204
-73449 90013
-53311 103301
-51436 99374
-57354 90529
-42432 96559
-53083 93836
-46623 88385
-68747 90693
-51757 82638
-42231 88308
-52011 86828
-62984 101048
-54848 95394
-71238 96771
-56914 97575
-73329 86947
-48414 89181
-73098 95359
-54050 110560
-49894 88487
-66302 85282
-67065 98932
-64818 84109
-46131 96956
-73127 101074
-63405 90053
-73266 101192
-53938 96881
-70905 107916
-67197 106129
-72525 95589
-70629 107999
-73382 82292
-58356 93378
-72966 87664
-64802 94136
-56962 106361
-46039 96289
-70141 91014
-57507 95052
-72487 98112
-47307 82105
-58498 92560
-50465 99681
-59921 93166
-52390 83852
-60757 111257
-43628 91383
-63217 99228
-41422 109693
-65717 111043
-49762 83899
-53977 104795
-73385 88346
-57870 105367
-54609 99966
-73231 101548
-57958 88989
-56257 94131
-57092 97399
-46811 89535
-46833 99890
-50663 83678
-45945 86060
-47108 111342
-47703 100037
-61855 86494
-47639 106221
-51156 103115
-53638 86212
-51272 91698
-45106 86978
-40911 99165
-64177 106123
-41724 86502
-71929 100760
-54233 98648
-48339 101772
-42992 103508
-54260 83822
-57873 100448
-62903 101764
-67767 87207
-70619 93131
-49727 110330
-50960 86290
-55731 100323
-43269 110974
-62580 83768
-61321 84974
-50652 109175
-62736 85097
-49157 89792
-57988 99023
-68407 99730
-46857 98036
-44620 93297
-66673 86900
-66505 81809
-63538 95587
-51421 92612
-55248 95925
-60344 89698
-68534 107305
-58553 103373
-66170 82890
-64532 84385
-63952 105034
-52541 93254
-70835 107226
-71740 104538
-70974 88492
-42109 107983
-47731 101716
-52678 104537
-57928 97376
-64747 96334
-55650 93888
-70376 104972
-60089 101544
-57930 93999
-65209 100017
-72893 98328
-46758 100722
-55521 101270
-48336 88245
-68608 92764
-49869 93597
-69142 99083
-68781 109272
-66936 94445
-62031 95916
-43182 93389
-69034 87117
-56597 107486
-63965 85931
-52799 88615
-42057 83941
-66990 101117
-60030 86686
-49123 98272
-68924 90213
-71386 109372
-46052 92890
-46702 87995
-50286 82920
-54040 87156
-48533 108959
-58181 88154
-72671 105930
-52957 103455
-60199 110771
-69162 101058
-62001 83361
-68723 93217
-49604 93095
-61435 88753
-51687 106669
-43793 83470
-46920 106533
-46024 109226
-61805 108231
-57388 81950
-60354 89092
-69193 110824
-49018 96685
-44154 103383
-66490 99209
-46008 88121
-62416 109066
-42880 107610
-51219 95706
-50459 102268
-47430 104238
-61924 89906
-50955 106271
-45740 104371
-66112 110888
-49287 104837
-43456 110224
-52754 103136
-64539 84139
-50193 82329
-62178 93959
-41813 90487
-51899 93058
-62836 108790
-46373 98661
-60499 92654
-59511 99175
-40789 95118
-72852 91737
-49265 106153
-65164 92365
-47870 107381
-49168 102983
-64879 95751
-47636 82133
-53908 93499
-59125 110956
-65252 108341
-65251 106965
-52700 98874
-72616 101538
-49043 106549
-43681 95096
-50010 95036
-67779 109649
-47373 108710
-42701 104115
-44675 87496
-67557 110937
-48329 88558
-65931 100251
-50124 84550
-53886 91056
-50419 98222
-66736 103956
-56537 92914
-69809 83873
-61371 95619
-57607 93847
-44933 82281
-59904 92854
-44450 83409
-57707 97799
-53467 92976
-71307 110068
-64666 95087
-43119 108242
-57861 94912
-44173 109474
-67059 100357
-53620 100773
-54687 106002
-41660 101791
-71938 91205
37818 -52682
43267 -61688
32554 -75784
21667 -63034
43044 -71905
22114 -55991
35588 -74470
43105 -72144
25576 -71919
19469 -74122
37059 -56326
47875 -83070
39854 -64723
25949 -64965
19163 -72254
19182 -57645
28610 -72752
23395 -66417
35117 -55715
31964 -51357
37145 -57457
41699 -60381
20131 -73821
30889 -76667
21296 -72390
25262 -68892
38084 -61321
36320 -75969
39673 -76867
32655 -70916
31626 -51970
46393 -51830
26901 -68221
37547 -69369
21776 -75789
38674 -78606
24633 -65943
41729 -80575
46702 -67538
19164 -54753
37677 -80842
34553 -65358
23078 -72129
38266 -67297
33283 -58852
47504 -80712
34218 -67233
32606 -56250
40986 -77319
27849 -60702
39972 -75824
34861 -74309
44288 -66942
32400 -82976
32165 -66179
20837 -55820
42502 -73230
21545 -54470
24994 -76655
38781 -57046
23364 -76527
46049 -80306
40816 -70868
35316 -58471
43698 -80628
26339 -62794
46362 -52619
31124 -58112
46155 -67002
32522 -76774
31106 -65066
19772 -71725
33938 -79150
40146 -59068
35001 -72667
30441 -74078
23254 -63004
30163 -67452
25759 -67256
24760 -60364
23593 -60972
28422 -77566
26027 -80187
44095 -64837
43768 -77293
33030 -72025
42308 -57730
36559 -76913
33848 -59380
33452 -71512
29328 -62151
39724 -53702
45593 -58627
33621 -69647
45124 -52068
23497 -75581
26358 -52152
36114 -60708
35061 -75541
37499 -54975
37672 -60247
25800 -59506
21805 -79646
29010 -82469
30059 -64760
26766 -73201
40619 -75111
25474 -62541
29241 -56364
45044 -76961
26640 -67025
36526 -67763
28898 -66111
25456 -57888
31186 -82403
33499 -56271
38988 -58214
21581 -66992
31221 -76462
20990 -56747
18578 -79090
44766 -71193
32239 -60030
34414 -82319
40621 -52285
19871 -74763
29530 -76126
46381 -53060
22810 -65784
43303 -63017
23873 -67297
42855 -54578
35476 -59329
46972 -69642
22016 -58360
37429 -73032
18209 -67886
27728 -81965
21037 -77610
22771 -71450
45592 -80398
47693 -59438
25401 -77029
30420 -75054
36544 -80315
29160 -72675
33274 -60648
23579 -58851
43421 -71868
47328 -77352
32575 -65486
21406 -75558
26524 -75684
18796 -78816
27030 -76144
45845 -79179
30407 -79288
26693 -68265
27554 -63386
34952 -61853
36685 -70053
41680 -81362
20721 -56399
24106 -65491
39117 -80969
30616 -76466
40951 -79330
47732 -81830
31127 -53019
44715 -66369
34819 -53687
23584 -59012
40595 -58539
20556 -80096
36757 -68439
43418 -57774
25943 -78883
30487 -69004
42970 -69250
47932 -81101
35142 -55185
20522 -72797
47568 -55124
40638 -62058
27957 -72755
22753 -82080
39920 -56932
33865 -54725
47757 -56999
31468 -74528
26565 -62353
22275 -53228
24764 -74226
18826 -60582
22323 -52043
47342 -55672
33796 -62838
18179 -71488
45926 -58250
27739 -71676
18033 -81453
41954 -80570
41002 -52623
35722 -54682
39165 -54892
29964 -60906
22175 -64767
47339 -55493
40127 -57188
37712 -52128
27455 -77995
34366 -60218
20243 -66408
22948 -65841
24252 -81621
42178 -61366
44823 -81996
18777 -75291
47018 -63668
29493 -53452
44916 -55042
35831 -72461
23291 -58240
20256 -82856
32823 -68067
23190 -67033
35693 -81285
18636 -60886
47534 -63420
44252 -60850
34246 -59615
18406 -60712
27346 -77800
27428 -79370
34974 -67191
21951 -62363
23851 -74523
31789 -57582
39592 -70685
47597 -61539
43327 -66702
25311 -54641
27136 -62012
32781 -51249
21924 -57661
41731 -68053
38942 -82003
25764 -81876
47260 -61204
41601 -64222
25500 -71316
47021 -64151
40029 -55281
46618 -82010
34481 -57381
47141 -67875
36117 -78276
28403 -79260
28027 -59620
39934 -53054
18089 -83086
46585 -81139
27190 -76542
23738 -61302
25880 -58290
21480 -65527
32635 -63408
18660 -54085
26441 -57936
35126 -62899
25420 -65745
39561 -55418
21522 -58345
40440 -73270
41151 -79563
21164 -74451
46929 -62098
44477 -63617
34235 -67985
27069 -70348
42315 -56406
44346 -57163
24265 -54453
42791 -70133
26558 -74509
32891 -70793
21965 -76713
23964 -79716
25667 -55668
23596 -67428
23665 -79767
21072 -68969
22800 -68942
25616 -57906
22322 -56858
22698 -80086
26754 -69756
38652 -81671
43745 -67426
28909 -71805
19389 -82683
47324 -71622
37601 -63332
33622 -51447
30384 -55961
46093 -82351
22422 -54603
38238 -65722
39908 -81871
44521 -68091
26236 -62780
40993 -51622
35248 -69967
25597 -79499
40422 -67266
46607 -54070
43537 -59140
36309 -69174
29642 -52149
45138 -73651
45680 -68681
29049 -67361
46444 -53500
20852 -65425
29307 -56003
35244 -68917
22157 -53189
34486 -72805
46998 -69373
28872 -51396
25788 -63925
38571 -79696
41511 -66179
47807 -61190
18468 -69753
37303 -58356
35849 -67880
32288 -70087
37057 -67552
46914 -61461
21179 -66063
36294 -68147
20160 -83064
46486 -66800
31629 -76768
33581 -60587
29905 -82256
41424 -73710
41065 -82771
41890 -70202
47510 -61837
40442 -53606
28540 -72719
38188 -77465
31272 -61373
23563 -66064
27526 -80958
33292 -70831
33104 -68790
23666 -53409
38434 -82421
35425 -65312
42557 -78697
44451 -55107
35557 -55455
39949 -69305
38207 -66224
33265 -59322
22235 -56814
45543 -53016
37840 -81376
18338 -67413
31086 -52878
24894 -57436
30707 -57761
36213 -79403
22071 -80191
18475 -60821
20894 -55003
28944 -62452
37275 -80387
29106 -77659
45475 -81653
39180 -64405
32833 -57121
46616 -70057
42086 -51550
34250 -60716
44001 -63201
23033 -63687
28820 -51930
36293 -68227
22310 -65318
38044 -56117
36581 -58947
34350 -54671
25603 -57596
34225 -65071
34134 -71427
30030 -59346
20316 -64104
31896 -74899
18140 -65727
41059 -67951
30592 -66285
27049 -68574
39493 -54269
22136 -72234
47719 -60562
20838 -57054
33891 -57044
32236 -75187
27141 -73960
22556 -60417
26635 -54060
28073 -66950
34607 -72712
37076 -76464
19842 -75338
38613 -57900
24397 -53039
42391 -60182
18133 -62356
34031 -52178
36702 -82332
37616 -81468
42732 -65540
27245 -81023
34981 -61510
24903 -67553
23907 -55354
24491 -64550
42649 -54973
21614 -56679
46103 -81073
24674 -66105
38321 -57062
25202 -52567
21093 -68664
41930 -67331
41632 -61191
41165 -66802
26591 -53954
29840 -60242
37759 -54013
47972 -81726
24186 -56879
39072 -54813
39279 -77026
44569 -80286
25825 -68786
31366 -61836
28314 -67293
42267 -65955
34368 -79841
19551 -74380
35222 -79511
32686 -51919
36894 -72920
41830 -68843
44996 -58522
32486 -58357
27195 -64988
31917 -72625
30514 -79564
27661 -80659
42830 -75954
35982 -78399
22755 -73539
38276 -76546
43791 -55007
19987 -77066
44044 -67371
37343 -57834
37650 -81627
32044 -73246
33540 -67583
25389 -78472
27314 -75322
23443 -67051
36425 -62163
26076 -51754
22633 -81880
27744 -55966
36967 -80119
44194 -57206
25654 -73065
27152 -61505
47880 -78120
18697 -74853
46326 -66554
26217 -61845
26336 -59150
30097 -60371
30797 -77668
28289 -65108
32470 -74119
18592 -78385
28219 -69116
23605 -81912
19984 -73953
39722 -82322
21779 -63554
26973 -67168
33684 -81122
46955 -63108
26208 -62853
38485 -81515
30391 -60542
27860 -54046
29991 -82218
45885 -51985
19582 -55264
38947 -67176
20503 -74981
40157 -63317
24770 -79346
40860 -55751
40270 -81952
39040 -53323
43829 -75280
24476 -63583
44277 -67949
27709 -61438
46332 -60292
47983 -77823
24722 -63315
42799 -53347
47747 -79719
34100 -74063
24018 -58357
45146 -60972
45334 -64067
26587 -54786
45980 -74337
24147 -63832
35516 -65050
21083 -52435
37159 -64136
22765 -61017
32321 -64401
22434 -57940
42243 -60853
27318 -79530
44189 -75668
29348 -66644
32348 -63391
40789 -56931
33860 -62856
46111 -73676
24494 -57967
32713 -73531
38267 -64234
46706 -59592
36884 -68698
31935 -62954
31225 -68112
28378 -82762
20779 -63160
41670 -52820
32223 -57869
40574 -68022
27721 -77360
25726 -61504
43760 -71771
34542 -61079
25444 -71825
35942 -55854
48069 -80784
28750 -68929
39966 -81600
37484 -56044
21823 -62517
23958 -67198
36370 -75235
43516 -82468
45353 -64333
46556 -67165
44911 -70197
19023 -65846
35562 -78564
27343 -69374
27105 -67945
36572 -81643
47357 -75456
47295 -53000
28451 -73896
44592 -72064
29003 -70542
40362 -52422
47763 -78011
31781 -51945
30503 -77922
33410 -56838
45109 -56595
32471 -70968
42410 -67676
36971 -67150
24890 -66471
31965 -82823
37107 -55093
47108 -60811
38955 -67823
20925 -51883
25815 -53092
39896 -63179
27813 -61976
45899 -72429
27064 -70811
18460 -75708
39373 -55995
35889 -81193
28125 -61212
23192 -63078
39301 -77703
27375 -82408
47522 -59509
23674 -52051
28915 -79396
26861 -62198
41020 -55448
39008 -74552
36416 -79160
41201 -70891
25645 -52405
29187 -82832
34327 -69966
37708 -57155
23564 -72081
44954 -64023
24520 -69847
25447 -64700
22250 -71159
29030 -57565
43403 -71551
29774 -69951
44991 -63944
28661 -68473
36352 -65492
48001 -51756
27321 -76161
34273 -74513
22460 -59668
20111 -82909
29118 -72644
39179 -60772
42051 -55704
31648 -67347
27547 -52782
23621 -78108
47562 -67766
19725 -67909
45427 -79364
18479 -67063
22780 -59813
24325 -71643
32842 -68134
40330 -66986
31583 -73241
35802 -52033
35309 -54344
32487 -61915
35727 -75121
24557 -76758
45978 -72106
40057 -65768
23501 -54532
41194 -59334
21555 -81877
45217 -61851
22368 -74514
30306 -61680
43050 -55769
30791 -59393
41243 -61704
44036 -58813
26660 -67788
19200 -77429
31300 -82740
45028 -52735
35087 -62210
23300 -65227
47954 -58428
42020 -60511
25583 -73440
40352 -57596
21578 -54485
47091 -66945
34878 -61354
31099 -74507
22022 -57301
25540 -58344
25803 -74900
30654 -77627
27892 -66865
20739 -75542
45118 -81392
41436 -64100
40889 -58432
18286 -76012
23846 -61052
38326 -81061
21076 -53661
45632 -56692
24191 -69018
45485 -60862
40960 -79000
44696 -68871
19849 -55363
31671 -67544
42278 -66055
26041 -70392
45202 -81316
46402 -72570
33931 -75957
26421 -68589
39730 -69818
45453 -79913
44181 -55301
23338 -73653
47655 -57393
31853 -60115
41895 -76497
43260 -53760
47281 -59360
27888 -72217
38020 -78834
47816 -51845
40044 -82857
40865 -72184
32451 -71004
45623 -77924
25959 -64640
22148 -53173
45584 -78318
29221 -53716
28109 -56604
40767 -51208
28829 -51670
31037 -71583
29372 -82216
35025 -76285
47699 -76945
43920 -64247
43816 -57361
22716 -67892
26404 -56631
25661 -78350
45040 -70706
27951 -64596
38586 -64101
31875 -70447
23868 -67806
20175 -60306
35057 -61580
34238 -52251
30771 -71312
23353 -66382
41087 -55973
25406 -77390
32663 -62519
46392 -78131
19333 -82210
36238 -52440
45211 -63828
46959 -75548
33213 -71940
29732 -51240
28504 -65902
24634 -73557
46789 -54926
28211 -77452
43293 -54260
38145 -52601
18333 -56341
41041 -81023
37767 -65982
39405 -82337
39843 -58826
45762 -77762
34175 -79267
33546 -69987
45751 -67423
33098 -69944
26513 -60913
28427 -75956
31112 -74225
42181 -66797
35911 -70625
23150 -69588
21831 -61077
34887 -68057
25276 -55416
36451 -79782
42472 -63013
35951 -65800
46740 -76635
44378 -68824
39252 -70555
45458 -72715
19327 -59547
18833 -71365
43630 -71902
29627 -76754
30896 -81705
40485 -52195
40451 -61593
36203 -59730
24284 -71176
41677 -54383
42085 -55490
20499 -76193
39618 -68667
44057 -67552
24538 -82409
31755 -70358
37364 -76896
18247 -75697
39653 -72030
47758 -79461
40807 -59848
24582 -64500
32489 -72347
24136 -54169
24324 -51239
36726 -70317
43881 -73653
19901 -68684
41656 -80696
41367 -52043
35703 -64825
46593 -71929
44011 -72982
46626 -70965
28801 -80432
28853 -67693
31395 -56609
29533 -75534
22893 -53938
21040 -62462
31549 -68515
18163 -75749
21617 -66303
21211 -74679
33669 -67057
37506 -78590
40920 -67291
18719 -59162
42029 -60776
27504 -65473
45570 -67832
38480 -51971
38228 -63068
36040 -71033
41743 -60380
29925 -64243
23483 -78857
30806 -74003
18593 -53844
35929 -56399
26004 -59413
28227 -74569
29394 -76213
43691 -59254
39729 -78831
48029 -79415
20987 -67233
35119 -74606
41109 -51214
28233 -70168
45220 -53920
25249 -56263
34122 -69458
33779 -53420
34991 -74273
27336 -73438
38026 -57665
18828 -60303
38911 -66870
47349 -56676
46678 -76509
42184 -76885
23799 -64504
20169 -67176
38891 -74646
29526 -60357
43021 -83060
40152 -70932
24053 -75722
19346 -57613
21105 -72391
36566 -54711
18614 -67736
37845 -69327
31171 -64164
40050 -77829
45363 -73090
31561 -52675
25034 -61246
47256 -59700
34064 -80614
41435 -57693
23265 -57948
21087 -70119
22255 -51451
43786 -60810
43960 -54940
30282 -65672
22400 -56329
44883 -61244
36556 -77654
36789 -77978
40400 -78107
36254 -74912
24254 -66986
35227 -57626
29927 -82921
30218 -55406
28925 -65419
47792 -68352
38575 -82454
35681 -73556
39137 -69917
29021 -81577
38302 -54988
37300 -54361
19276 -80228
31261 -67544
22268 -74236
22724 -52245
41662 -74490
39128 -67213
41562 -62182
20671 -54271
24904 -57298
19255 -82408
43304 -80989
20076 -76081
25830 -65065
32002 -55633
40365 -53747
18359 -75997
45143 -61426
