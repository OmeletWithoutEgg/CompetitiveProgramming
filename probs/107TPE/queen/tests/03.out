-11353.69 13791.71
